guides/source/developers/extensions/testing-extensions.html.md

```diff
@@ -7,47 +7,67 @@ Solidus extensions.
 
 [contrib]: https://github.com/solidusio-contrib
 
-## TravisCI
+## CircleCI
 
-We usually test our extensions on TravisCI and make use of their [build matrix
-feature][build-matrix] to test across multiple Solidus versions. We also test
-against multiple databases: MySQL and PostgreSQL.
+We usually test our extensions against multiple Solidus versions and databases:
+MySQL and PostgreSQL. We use a [CircleCI Orb][orb] to re-use a maintain a single
+configuration that will be executed against all Solidus extensions.
 
-Solidus extensions should be tested across all versions of Solidus that have not
-reached [End of Life](https://solidus.io/blog/2018/01/04/maintenance-eol-policy.html)
-including the `master` branch.
+This Orb will take care of testing extensions against all versions of Solidus
+that have not reached [End of Life][eol] including the `master` branch.
 
-Here's an example `.travis.yml` testing versions 2.2 through 2.7 and the
-`master` branch:
+Here's an example `.circleci/config.yml` file that you should have in your
+extension:
 
 ```yaml
-language: ruby
-rvm:
-  - 2.3.1
-env:
-  matrix:
-    - SOLIDUS_BRANCH=v2.2 DB=postgres
-    - SOLIDUS_BRANCH=v2.3 DB=postgres
-    - SOLIDUS_BRANCH=v2.4 DB=postgres
-    - SOLIDUS_BRANCH=v2.5 DB=postgres
-    - SOLIDUS_BRANCH=v2.6 DB=postgres
-    - SOLIDUS_BRANCH=v2.7 DB=postgres
-    - SOLIDUS_BRANCH=master DB=postgres
-    - SOLIDUS_BRANCH=v2.2 DB=mysql
-    - SOLIDUS_BRANCH=v2.3 DB=mysql
-    - SOLIDUS_BRANCH=v2.4 DB=mysql
-    - SOLIDUS_BRANCH=v2.5 DB=mysql
-    - SOLIDUS_BRANCH=v2.6 DB=mysql
-    - SOLIDUS_BRANCH=v2.7 DB=mysql
-    - SOLIDUS_BRANCH=master DB=mysql
+version: 2.1
+
+orbs:
+  # Always take the latest version of the orb, this allows us to
+  # run specs against Solidus supported versions only without the need
+  # to change this configuration every time a Solidus version is released
+  # or goes EOL.
+  solidusio_extensions: solidusio/extensions@volatile
+
+jobs:
+  run-specs-with-postgres:
+    executor: solidusio_extensions/postgres
+    steps:
+      - solidusio_extensions/run-tests
+  run-specs-with-mysql:
+    executor: solidusio_extensions/mysql
+    steps:
+      - solidusio_extensions/run-tests
+
+workflows:
+  "Run specs on supported Solidus versions":
+    jobs:
+      - run-specs-with-postgres
+      - run-specs-with-mysql
+  "Weekly run specs against master":
+    triggers:
+      - schedule:
+          cron: "0 0 * * 4" # every Thursday
+          filters:
+            branches:
+              only:
+                - master
+    jobs:
+      - run-specs-with-postgres
+      - run-specs-with-mysql
 ```
 
-[build-matrix]: https://docs.travis-ci.com/user/customizing-the-build/#Build-Matrix
+It run specs against all supported versions, databases and setup a weekly
+nightly build to be sure the extension works even if it's changed in a while,
```

Comment:
`even if it's changed in a while` I would change the words a bit, to something like `even when it has not been updated for a while`